lib/ansible/modules/system/user.py

```diff
@@ -514,7 +526,13 @@ def modify_user_usermod(self):
 
         if self.expires:
             cmd.append('--expiredate')
-            cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+            try:
+                if self.clearexpires < 0:
```

Comment:
Hrm, `None > 0` will trigger this case, so won't any non-numeric date value do this?